wallet/src/Pos/Wallet/Web/Pending/Types.hs

```diff
@@ -115,7 +115,7 @@ instance Buildable (SecureLog PtxCondition) where
 data PtxSubmitTiming = PtxSubmitTiming
     { _pstNextSlot  :: SlotId
     , _pstNextDelay :: FlatSlotId
-    } deriving (Eq, Show)
+    } deriving (Eq, Show, Generic)
```

Comment:
Why we need the `Generic` instance here?

Reply:
For the `Arbitrary` instance use using `genericArbitrary`, and because it's a very useful instance in _general_.